docs/release_notes.rst

```diff
@@ -66,6 +66,7 @@ Bug Fixes
 
 Bug Fixes
 
+* Update convention support documentation (#386, #393)
```

Comment:
Looks like you updated the old release notes. Can you add it to the current development version?

Reply:
Done. In hindsight I am surprised that wasn't part of the merge conflict.